specification/implementation/metadata/mets/structmap/index.md

```diff
@@ -9,6 +9,8 @@ repeating the structMap element.
 The most crucial requirements for the CSIP mandated structural map are as follows:
 
 - The structMap element has a mandatory attribute @LABEL which has the fixed value of “CSIP structMap”. The @LABEL attribute is used to distinguish the Common Specification mandated structural map occurrence from any other, user-defined, structural maps. As such we can also derive the requirement, that any user-defined structural maps must not use the @LABEL value of “CSIP structMap”;
+- The structure is expressed with division elements (div). The div element can be nested in a tree structure and the implementation used in CSIP gathers the div elements described below in one main structural division element. 
+  - The main structural division use the attribute LABEL with the value being the string value of the content identification element of the package
```

Comment:
This top level structural division must use the package identifier as the value of its `@LABEL` attribute.